README.md

```diff
@@ -13,7 +13,7 @@ Install
 This package can be install through `pip` using
 
 ```
-$ pip install https://github.com/benchopt/benchopt
+$ pip install git+https://github.com/benchopt/benchopt
```

Comment:
actually I would prefer

pip install -U https://api.github.com/repos/benchopt/benchOpt/zipball/master

as it does not require to have git on the machine.